### PR TITLE
Add option hidden to function toJSON()

### PR DESCRIPTION
**Pull request by jsomariano**
Hi friends, how are you?

The bookshelf support this, link: https://bookshelfjs.org/api.html#Model-instance-toJSON

Please fill in this template.

I change the package @types/bookshelf, add the pending field hidden to function toJSON of model, very simple.

### CODE REVIEW

File: types/bookshelf/index.d.ts [hunk 1]
```diff
@@ -373,6 +373,7 @@ declare namespace Bookshelf {
     interface SerializeOptions {
         shallow?: boolean | undefined;
         omitPivot?: boolean | undefined;
+        hidden: array | undefined;
```
Comment (RyWilliams) [inline types/bookshelf/index.d.ts:376]: I think this should be optional:
```suggestion
        hidden?: array | undefined;
```

Comment (jsomariano) [reply types/bookshelf/index.d.ts]: Good, ty!

File: types/bookshelf/index.d.ts [hunk 2]
```diff
@@ -373,6 +373,7 @@ declare namespace Bookshelf {
     interface SerializeOptions {
         shallow?: boolean | undefined;
         omitPivot?: boolean | undefined;
+        hidden?: array | undefined;
```
Comment (ianldgs) [inline types/bookshelf/index.d.ts:376]: Shouldn't it be:

```suggestion
        hidden?: string[];
```

?

**Comment by ianldgs** [comment]
I'm happy to +1 if you make CI pass

**Pull request closed without merging**